docs/server-request.md

```diff
@@ -17,20 +17,38 @@ A low-level action that allows for any kind of HTTP request to be performed.
 - Read files or binary streams
 - Call APIs from same app system - use the [Call Local API](/docs/Actions/CallLocalApi.md) action instead
 
-## `Related Actions`
+## `Related actions`
 
 - [Parse JSON](/docs/Actions/CallApi.md) - creates variables from JSON returned by API calls
 - [Regex](/docs/Actions/CallApi.md) - create variables by matching patterns inside the content
 
-## `Parameter Reference`
+## `Parameter reference`
 
-| Parameter     | Description                           | Supports Variables |
+| Parameter     | Description                           | Supports variables |
 |---------------|---------------------------------------|--------------------|
 | URL           | Absolute URL where the resource to be invoked lives. | Yes |
-| HTTP Method   | The operations to perform against the URL.<br/> Possible values are GET, POST, PUT, DELETE, HEAD. | No |
+| Enforce SSL   | This will for ce request to be sent to a secure version of the URL.(https instead of http)| No |
```

Comment:
In English, always put the dot after the parenthesis (like this).

Reply:
Also, how does this affect the URL above? 
If it's http:// will it be converted to https://? 
Why would people not just put https to start with? 
What problem does it actually try to solve? 
How it affects relative URLs?

Reply:
What is for ce?
(Have you already installed a spell check plugin?)

Reply:
URLs can also come from tokens, which may use http or https. The option is there to make sure no matter what the token brings, it uses a secure versions.
In regards to relative URLs see below.

Reply:
Yeah, maybe say that in the documentation, otherwise it feels like a useless setting.